tests/registry-test-case.php

```diff
@@ -12,50 +13,48 @@
 class RegistryTestCase extends WP_UnitTestCase {
 	use ArraySubsetAsserts;
 
-	protected $registry;
-	protected $globally_registered_blocks = [];
-
-	protected function setUp(): void {
-		parent::setUp();
+	protected function tearDown(): void {
+		// Unregister non-core blocks.
+		$block_registry = WP_Block_Type_Registry::get_instance();
```

Comment:
This is a clever way to persist block registrations during GraphQL tests, and also support core blocks for testing. Nice!